src/client/auth/oidc.rs

```diff
@@ -325,14 +347,49 @@ async fn send_sasl_start_command(
     send_sasl_command(conn, sasl_start).await
 }
 
-async fn do_two_step_auth(
+// do_shared_flow is the shared flow for both human and machine
```

Comment:
Name change reflects that this is no longer necessarily a two step sasl communication.